systemd/systemd_test.go

```diff
@@ -1356,6 +1405,88 @@ func (s *SystemdTestSuite) TestUnmaskInEmulationMode(c *C) {
 		{"--root", "/path", "unmask", "foo"}})
 }
 
+func (s *SystemdTestSuite) TestListMountUnitsEmpty(c *C) {
+	s.outs = [][]byte{
+		[]byte("\n"),
+	}
+
+	sysd := New(SystemMode, nil)
+	units, err := sysd.ListMountUnits("some-snap", "")
+	c.Check(units, IsNil)
+	c.Check(err, IsNil)
+}
+
+func (s *SystemdTestSuite) TestListMountUnitsMalformed(c *C) {
+	s.outs = [][]byte{
+		[]byte(`Description=Mount unit for some-snap, revision x1
+Where=/somewhere/here
+FragmentPath=/etc/systemd/system/somewhere-here.mount
+HereIsOneLineWithoutAnEqualSign
+`),
+	}
+
+	sysd := New(SystemMode, nil)
+	units, err := sysd.ListMountUnits("some-snap", "")
+	c.Check(units, IsNil)
```

Comment:
in error cases usually I will omit the first argument and just check that the error is what we expect it to be, but if you want to keep this case I would suggest changing it like above to be HasLen based

Reply:
Done.